### ISSUE TITLE
Add './_pginfo/**/*.{html,json}' to content of tailwind.config.js

### ISSUE THREAD

**Issue by TechAkayy** [domsinclair/PinegrowBits#2]
Hi Dom, 

Awesome and very helpful repo. 

Can I suggest adding './_pginfo/**/*.{html,json}' to the content of tailwind.config.js in your basic site tutorial.

https://github.com/domsinclair/PinegrowBits/blob/master/basicsite.md

Thanks bunch!

**Comment by TechAkayy** [comment]
Ahh okay... You are covering that in the subsequent one - https://github.com/domsinclair/PinegrowBits/blob/master/pgbasictemplate.md. Gotcha :-)

**Comment by domsinclair** [comment]
As you've probably worked out you've inadvertently stumbled across a small but not insignificant fault with the approach that I took to all of this.

I find one of the easiest ways to learn something new is to document it as I go along.  Each tutorial therefore is really. in truth, my taking a new step in my own learning curve.

So arguably the quickest an easiest way I've found to configure a new PG / tailwind project is in the last tutorial here on creating a custom library.  That will no doubt remain the case until I find a better way still and I'm sure there must be one!
